src/snowflake/snowpark/udf.py

```diff
@@ -897,3 +915,115 @@ def _do_register_udf(
                 )
 
         return UserDefinedFunction(func, return_type, input_types, udf_name)
```

Comment:
Again, just edit the code. This instantiation is broken now because you changed the ctor. You're using git, no need to keep old dead code lying around.